templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -119,6 +119,15 @@ Parameters:
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
       It cannot start or end with a hyphen (-).
     Type: String
+  QSS3BucketRegion:
+    Default: 'us-east-1'
+    AllowedPattern: ^[0-9a-z]+([0-9a-z-]*[0-9a-z])*$
+    ConstraintDescription: Quick Start bucket region can include numbers, lowercase
+      letters, and hyphens (-). It cannot start or end with a hyphen
+      (-).
+    Description: The AWS Region where the Quick Start S3 bucket (QSS3BucketName) is
```

Comment:
I think we might need to change the description of this parameter so that it's more clear to customers that it's not required for typical deployments

Reply:
This from what I remember (speaking with Tony) is a mandatory value, defaulting to `us-east-1`... I'm going to add Tony to these reviews so we can agree on this stuff earlier in the pipeline.

Reply:
I'll pull @ddomingorh  in on this.